packages/transitive-overlay/src/index.tsx

```diff
@@ -129,9 +133,43 @@ const TransitiveCanvasOverlay = ({
   if (showRouteArrows) {
     mapImages.push({
       id: "arrow-icon",
-      url: routeArrow
+      url: routeArrow,
+      options: { sdf: true }
     });
   }
+  mapImages.push({
+    id: "circle-1",
+    url: circle,
+    options: {
+      sdf: true,
+      content: [440, 370, 560, 630]
```

Comment:
Is this the content box? Add a comment to compute this. If padding goes in here, then a prop of type `number[4]` would probably do.